api_specs/v3/component_analyses_v3.yaml

```diff
@@ -0,0 +1,413 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: '3.1'
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Dependency Analytics Server"
+security:
+- tokenAuth: []
+paths:
+  /component-vulnerability-analyses/:
+    post:
+      tags:
+        - CA batch call
+      summary: Get vulnerability details about a software component and recommendations to change to a more secured version in a batched manner.
+      parameters:
+        - name: uuid
+          in: header
+          description: User Identification uuid
+          required: false
+          schema:
+            type: string
+        - name: X-Request-ID
+          in: header
+          description: Unique id for every CA call
+          required: false
+          schema:
+            type: string
+        - name: X-Telemetry-ID
+          in: header
+          description: Unique RedHat id
+          required: false
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RequestPackageDetails'
+
+      responses:
+        200:
+          description: Detailed response for the requested application component batch
+          headers:
+            uuid:
+              schema:
+                type: string
+              description: User uuid.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  anyOf:
+                    - $ref: '#/components/schemas/FreeUserBatch'
+                    - $ref: '#/components/schemas/NoVulnerabilities'
```

Comment:
I dont feel this is needed. Do we need to send blank response for packages which doesnt have any vuln?
Same for GET operation as well

Reply:
Resolved.

Reply:
Resolved.